llabake/Hello-Books
# Ft build homepage with react 153906223

#### What does this PR do?
This PR uses react to create the home page
#### Description of Task to be completed?
User should be able to see a home page
#### How should this be manually tested?
* clone the repo
* cd into the project directory
* Run npm install to install all the dependency
* Run npm run dev
* Visit the port where the webpack-dev-server started, the home page should be loaded there
#### What are the relevant pivotal tracker stories?
153906223
#### Any background context you want to add?
N/A
#### Screenshots
![image](https://user-images.githubusercontent.com/29184825/34390016-4c702df4-eb3d-11e7-8ae2-a537423d540f.png)
